docs/root/intro/arch_overview/service_discovery.rst

```diff
@@ -91,6 +88,20 @@ Generally active health checking is used in conjunction with the eventually cons
 discovery service data to making load balancing and routing decisions. This is discussed further in
 the following section.
 
+.. _arch_overview_service_discovery_types_sds:
+
+Service discovery service (SDS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+*(deprecated)* This discovery service was replaced to endpoint discovery service in :ref:`v2 xDS
+API<config_overview_v2>`.
+
+The *service discovery service* is a generic :ref:`REST based API <config_cluster_manager_sds_api>`
+used by Envoy to fetch cluster members. Lyft provides a reference implementation via the Python
+`discovery service <https://github.com/lyft/discovery>`_. That implementation uses AWS DynamoDB as
+the backing store, however the API is simple enough that it could easily be implemented on top of a
+variety of different backing stores.
```

Comment:
Can you add a RST warning or notice box that explains that SDS now is also used as an acronym for Secret Discovery service in v2? Thanks for helping sort out the confusion here.

Reply:
That's nice. How about this one? 0671030